frontend/src/about/about.js

```diff
@@ -103,7 +109,42 @@ class About extends React.Component {
                     </div>
                     <div>
                         <p>
-                            Copy about the project will go here.
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
Please open a follow-up issue to make sure we don't forget to fill this in with real text soon!